mltrace/db/migrations/versions/017c5f7dc0aa_add_git_tags_column.py

```diff
@@ -0,0 +1,24 @@
+"""Add git tags column
+
+Revision ID: 017c5f7dc0aa
+Revises: 799660f69710
+Create Date: 2021-06-30 14:28:42.827950
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '017c5f7dc0aa'
+down_revision = '799660f69710'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("component_runs", sa.Column("git_tags", sa.String))
```

Comment:
You can use the `PickleType` column instead of `String` to store lists. This way you don't have to convert back and forth from string to list. See an example in the `stale` column in `component_runs`

Reply:
updated to use PickleType instead of String

Reply:
great!